lib/dynamic_config/dcdo.rb

```diff
@@ -45,6 +45,14 @@ def frontend_config
       hoc_mode: DCDO.get('hoc_mode', false),
     }
   end
+
+  def to_h
+    @datastore_cache.all
+  end
+
+  def refresh
+    @datastore_cache.update_local_cache
+  end
```

Comment:
nit; can these helper methods be defined on `dynamic_config_base` instead? Seems like they apply to Gatekeeper just as well as DCDO

Reply:
Yes! That's definitely where they belong.